scc/git.py

```diff
@@ -959,6 +964,9 @@ def is_whitelisted_comment(x):
         pr_attributes["user"] = [pullrequest_user.login]
         pr_attributes["pr"] = ['#' + str(pullrequest.get_number())]
 
+        milestone = pullrequest.get_milestone()
+        pr_attributes["milestone"] = [milestone.title, str(milestone.number)]
```

Comment:
Running this in dry-run mode against e.g. Bio-Formats, I get

```bsh
sbesson@Sebastiens-MacBook-Pro-3 bioformats % ../scc/venv/bin/scc merge --info develop
2024-06-24 15:10:52,854 [   scc.merge] INFO  Listing Pull Request(s) based on develop
2024-06-24 15:10:52,855 [   scc.merge] INFO  Including Pull Request(s) opened by any public member of the organization
2024-06-24 15:10:52,855 [   scc.merge] INFO  Including Pull Request(s) labelled as include or dependencies
2024-06-24 15:10:52,855 [   scc.merge] INFO  Excluding Pull Request(s) staged as draft
2024-06-24 15:10:52,855 [   scc.merge] INFO  Excluding Pull Request(s) labelled as exclude or breaking
Traceback (most recent call last):
  File "/Users/sbesson/Documents/GitHub/scc/venv/lib/python3.9/site-packages/scc/main.py", line 64, in entry_point
    main("scc", items=[
  File "/Users/sbesson/Documents/GitHub/scc/venv/lib/python3.9/site-packages/yaclifw/framework.py", line 188, in main
    ns.func(ns)
  File "/Users/sbesson/Documents/GitHub/scc/venv/lib/python3.9/site-packages/scc/git.py", line 3329, in __call__
    self.merge(args, self.main_repo)
  File "/Users/sbesson/Documents/GitHub/scc/venv/lib/python3.9/site-packages/scc/git.py", line 3354, in merge
    updated, merge_msg = main_repo.rmerge(
  File "/Users/sbesson/Documents/GitHub/scc/venv/lib/python3.9/site-packages/scc/git.py", line 1819, in rmerge
    merge_msg += self.origin.find_candidate_pulls(filters)
  File "/Users/sbesson/Documents/GitHub/scc/venv/lib/python3.9/site-packages/scc/git.py", line 929, in find_candidate_pulls
    include, exclude_reason = self.filter_pull(pullrequest, filters)
  File "/Users/sbesson/Documents/GitHub/scc/venv/lib/python3.9/site-packages/scc/git.py", line 968, in filter_pull
    pr_attributes["milestone"] = [milestone.title, str(milestone.number)]
AttributeError: 'NoneType' object has no attribute 'title'
```

Reply:
Should be fixed now